zanata-war/src/main/java/org/zanata/webtrans/server/rpc/ActivateWorkspaceHandler.java

```diff
@@ -132,7 +132,7 @@ public ActivateWorkspaceResult execute(ActivateWorkspaceAction action,
         boolean isProjectObsolete = isProjectIterationObsolete(project.getStatus(), projectIteration.getStatus());
         boolean hasWriteAccess = hasWritePermission(project, locale);
         boolean hasGlossaryUpdateAccess = hasGlossaryUpdatePermission();
-        boolean requireReview = projectIteration.getRequireTranslationReview();
+        boolean requireReview = true;
```

Comment:
Are we bypassing the value in project version setting? If the project version is set to not requiring review this may cause problems.